.github/workflows/dapr.yml

```diff
@@ -173,3 +175,64 @@ jobs:
             --body "${RELEASE_BODY}" \
             --prerelease true \
             ${RELEASE_ARTIFACT[*]}
+      - name: Save release version
+        run: |
+          mkdir -p ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+          echo ${REL_VERSION} > ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}/${{ env.DAPR_VERSION_ARTIFACT }}
+      - name: Package Helm chart
+        if: ${{ env.LATEST_RELEASE }} == "true"
+        env:
+              HELM_CHARTS_DIR: charts/dapr
+        run: |
+          mkdir -p ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+          helm package ${{ env.HELM_CHARTS_DIR }} --destination ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+      - name: Upload Helm charts package to artifacts
+        if: ${{ env.LATEST_RELEASE }} == "true"
+        uses: actions/upload-artifact@master
+        with:
+          name: dapr_helm_charts_package
+          path: ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+  # This job downloads the helm charts package artifact uploaded by the publish job,
+  # checks out the helm charts git hub pages repo and commits the latest version of
+  # helm charts package.
+  helmpublish:
+    name: Publish helm charts to Helm github pages repo
+    needs: publish
+    if: startswith(github.ref, 'refs/tags/v') and !contains(github.ref, '-rc.')
```

Comment:
@vinayada1 PR build runs this job --- https://github.com/dapr/dapr/actions/runs/164812266

it is failling. I thought this should be skipped. can you please look at it ?

Reply:
fixed